### PR TITLE
Mirror cypress image to ci registry

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

**Review by bipuladh — APPROVED** [review]
/lgtm

File: core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ocp [hunk 1]
```diff
@@ -6,3 +6,4 @@ registry.access.redhat.com/ubi7/ubi-minimal:latest registry.ci.openshift.org/ocp
 registry.access.redhat.com/ubi8/ubi-minimal:latest registry.ci.openshift.org/ocp/ubi-minimal:8
 registry.access.redhat.com/ubi7/python-36:latest registry.ci.openshift.org/ocp/ubi-python-36:7
 registry.access.redhat.com/ubi8/python-39:latest registry.ci.openshift.org/ocp/ubi-python-39:8
+quay.io/ocs-dev/cypress:latest registry.ci.openshift.org/ocp/cypress:latest
```
Comment (petr-muller) [inline core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ocp:9]: No, definitely not `ocp` namespace

**Comment by vbnrh** [comment]
Closing this PR for now as we will be pulling the image through quay.

**Pull request closed without merging**